src/settings/LoanHistory/LoanHistoryForm.test.js

```diff
@@ -0,0 +1,254 @@
+    'anonymize': 'ui-circulation.settings.loanHistory.anonymize',
+    'treat': 'ui-circulation.settings.loanHistory.treat',
+    'closedLoansFeesFines': 'ui-circulation.settings.loanHistory.closedLoansFeesFines',
+    'anonymizeFeesFines': 'ui-circulation.settings.loanHistory.anonymizeFeesFines',
+  };
+  const testIds = {
+    form: 'form',
+    submitButton: 'submitButton',
+    closedLoans: 'closedLoans',
+    closedLoansHeadline: 'closedLoansHeadline',
+    closedLoansFeesFinesHeadline: 'closedLoansFeesFinesHeadline',
+    loansFeefineSection: 'loansFeefineSection',
+  };
+  const testHandleSubmit = jest.fn();
+  const testGetState = jest.fn(() => ({
+    values: {
+      treatEnabled: false,
+    },
+  }));
+  const testDefaultProps = {
+    handleSubmit: testHandleSubmit,
+    form: {
+      getState: testGetState,
+    },
+    pristine: false,
+    submitting: false,
+  };
+
+  const getById = (id) => within(screen.getByTestId(id));
+
+  afterEach(() => {
+    Button.mockClear();
+    Pane.mockClear();
+    PaneFooter.mockClear();
+    AnonymizingTypeSelectContainer.mockClear();
+    FieldArray.mockClear();
+    testHandleSubmit.mockClear();
+  });
+
+  describe('with default props', () => {
+    beforeEach(() => {
+      render(
+        <LoanHistoryForm {...testDefaultProps} />
+      );
+    });
+
+    it('should render form', () => {
+      const form = screen.getByTestId(testIds.form);
+
+      expect(form).toBeVisible();
+      expect(form).toHaveAttribute('noValidate', '');
+    });
+
+    it('should submit form', () => {
+      expect(testHandleSubmit).toHaveBeenCalledTimes(0);
+
+      fireEvent.submit(screen.getByTestId(testIds.form));
+
+      expect(testHandleSubmit).toHaveBeenCalledTimes(1);
+    });
+
+    it('should render Pane component', () => {
+      expect(Pane).toHaveBeenCalledWith(expect.objectContaining({
+        defaultWidth: 'fill',
+        fluidContentWidth: true,
+        paneTitle: undefined,
+      }), {});
+    });
+
+    it('should render PaneFooter component', () => {
+      expect(PaneFooter).toHaveBeenCalled();
+    });
+
+    it('should render submit Button component', () => {
+      expect(Button).toHaveBeenCalledWith(expect.objectContaining({
+        type: 'submit',
+        disabled: false,
+      }), {});
+    });
+
+    it('should render submit Button component label', () => {
+      expect(getById(testIds.submitButton).getByText(labelIds.buttonSave)).toBeVisible();
+    });
+
+    it('should render closed loans label', () => {
+      expect(getById(testIds.closedLoansHeadline).getByText(labelIds.closedLoans)).toBeVisible();
+    });
+
+    it('should render anonymize label', () => {
+      expect(getById(testIds.closedLoans).getByText(labelIds.anonymize)).toBeVisible();
+    });
+
+    it('should render closed loans AnonymizingTypeSelectContainer component', () => {
+      expect(AnonymizingTypeSelectContainer).toHaveBeenNthCalledWith(1, expect.objectContaining({
```

Comment:
call order map for `AnonymizingTypeSelectContainer`

Reply:
I'd like to discuss it. I specially did not create map